docs/snippets/performance_chunking.py

```diff
@@ -0,0 +1,18 @@
+"""
+description: Chunking example
+tags: performance
+"""
+
+import dlt
+
+if __name__ == "__main__":
```

Comment:
when using this snippet in the docs, only the code inside the main clause is used and any asserts are stripped (we can add an additional marker to have certain lines removed, but possibly this is not necessary). Also you can decide which imports show up in the snippet by either placing them in the head or in the main clause.